### PR TITLE
ENDOC-156 Rework Getting Started landing pages to use the CLI.

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
@joewhite101 and @es-entando , I pushed this to staging. https://entando.github.io/entando-docs/ and
https://entando.github.io/entando-docs/next/docs/getting-started/. The exact form of the CLI command will change a bit still since we'll be a dedicated domain on it and we're still discussing whether to use source or direct curl (there are user input options with source) but at least you can check the basic changes.

File: vuepress/docs/next/docs/getting-started/README.md [hunk 1]
```diff
@@ -2,9 +2,28 @@
...
+   ```
+
+    2. Install Entando
+
+    ```sh
+    curl "https://raw.githubusercontent.com/entando/entando-cli/develop/hr/auto" |\
+    ENTANDO_RELEASE="v6.3.0-sprint12" ENTANDO_CLI_VERSION="develop" bash
+   ```
+
+   3. The progress of the install will be displayed on the console and can take 10 minutes or so depending on your network bandwidth. Once complete, the installer will give you the URL to access the <span>Entando App Builder</span>, e.g. `https://quickstart-entando.192.168.64.33.nip.io/app-builder/`. Login with username:`admin`, password: `adminadmin`.
```
Comment (joewhite101) [inline vuepress/docs/next/docs/getting-started/README.md:23]: I wonder if "...depending on the time to download the docker images" is better than "network bandwidth"

Comment (joewhite101) [inline vuepress/docs/next/docs/getting-started/README.md:23]: Is the IP here guaranteed?

File: vuepress/docs/.vuepress/components/SpecialLayout.vue [hunk 2]
```diff
@@ -215,80 +215,37 @@
...
               <div class="instruction">
-                sudo kubectl get ingress -n entando -o jsonpath='{.items[2].spec.rules[*].host}{.items[2].spec.rules[*].http.paths[2].path}{"\n"}'
+                curl "https://raw.githubusercontent.com/entando/entando-cli/develop/hr/auto" | ENTANDO_RELEASE="v6.3.0-sprint12" ENTANDO_CLI_VERSION="develop" bash
               </div>
-              <p>Login to the <span>Entando App Builder</span> with username:<span>admin</span>, password: <span>adminadmin</span></p>
-              <p>See the <a href="/v6.2/docs/">Docs</a> and <a href="/v6.2/tutorials/">Tutorials</a> to continue your journey with Entando.</p>
+              <p>The progress of the install will be displayed on the console and can take 10 minutes or so depending on your network bandwidth.</p>
+              <p>See the <a href="/v6.2/docs/getting-started/">in-depth guide</a> to better understand the steps, or to perform them manually if you encounter issues.</p>
+              <p>Once complete, the installer will give you the URL to access the <span>Entando App Builder</span>, e.g. <span>https://quickstart-entando.192.168.64.33.nip.io/app-builder/</span>.
```
Comment (joewhite101) [inline vuepress/docs/.vuepress/components/SpecialLayout.vue:242]: Is this IP guaranteed?

**Review by joewhite101 — COMMENTED** [review]
Some questions

File: vuepress/docs/next/docs/getting-started/README.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/README.md:23]: I like that better. I'll update it when I get back to it.

Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/README.md:23]: Nope, it's meant to be an example of the URL, like we do elsewhere when we mention IPs.

File: vuepress/docs/.vuepress/components/SpecialLayout.vue (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/.vuepress/components/SpecialLayout.vue:242]: Same as above but maybe I can improve the language.

File: vuepress/docs/next/docs/getting-started/README.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/README.md]: Removed the URL since I don't think it's useful with the CLI in play.

Comment (nshaw) [reply vuepress/docs/next/docs/getting-started/README.md]: Updated

File: vuepress/docs/.vuepress/components/SpecialLayout.vue (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/.vuepress/components/SpecialLayout.vue]: Refined a bit.

**Comment by nshaw** [comment]
@joewhite101 and @es-entando, this is visible on my own staging copy now - https://nshaw.github.io/next/docs/getting-started/

**Review by joewhite101 — APPROVED** [review]
LGTM

**Pull request merged**